### ISSUE TITLE
Unable to install PyPcode 1.1.0

### ISSUE THREAD

**Issue by DarkaMaul** [angr/pypcode#44]
### Description

With the new pip release, using the package raises an error:

```command
(tmp-900339b97de62dc) ➜ /tmp python -c 'import pypcode'                         
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "./tmp-900339b97de62dc/lib/python3.10/site-packages/pypcode/__init__.py", line 13, in <module>
    from ._csleigh import ffi
ModuleNotFoundError: No module named 'pypcode._csleigh'
```

### Steps to reproduce the bug

1. Create a virtualenv for a fresh Python version (tested with 3.9, 3.10)
```command
$ mktmpenv -p $(which python3.10)
```
2.  ` pip install 'pypcode'`
3. `python -c 'import pypcode'`

However, the same steps with PyPcode 1.0.7 works:
```
$ pip install 'pypcode==1.0.7' 
$ python -c 'import pypcode'  && echo "ok"
```

### Environment

Environment:

* Linux (Debian Stable)
* Python 3.{9,10}
* PyPcode: 1.1.0

**Comment by DarkaMaul** [comment]
First detected by `Quokka` CI ( [link](https://github.com/quarkslab/quokka/actions/runs/3503775612/jobs/5869039722))

**Comment by r4dr3fr4d** [comment]
I'm seeing this one as well.

**Comment by mborgerson** [comment]
I will fix it asap

**Comment by r4dr3fr4d** [comment]
It seems to work okay on main/master

**Comment by mborgerson** [comment]
Issue should be fixed with new release v1.1.1 now on PyPI. Thanks for reporting
